<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-10</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>-10</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>-10</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>-10</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>                 Result</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Рассчитать СДНФ</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>И</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ИЛИ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Исключающее ИЛИ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Импликация</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эквивалентность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Штрих Шеффера</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Стрелка пирса</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>И</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ИЛИ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Исключающее ИЛИ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Импликация</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эквивалентность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Штрих Шеффера</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Стрелка пирса</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>И</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ИЛИ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Исключающее ИЛИ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Импликация</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эквивалентность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Штрих Шеффера</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Стрелка пирса</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>710</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Генерация таблицы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>710</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Генерировать схему</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>710</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Упростить 
СДНФ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
